svm/examples/json-rpc/server/src/rpc_process.rs

```diff
@@ -423,10 +423,7 @@ impl JsonRpcRequestProcessor {
         _error_counters: &mut TransactionErrorMetrics,
     ) -> TransactionCheckResult {
         /* for now just return defaults */
-        Ok(CheckedTransactionDetails {
-            nonce: None,
-            lamports_per_signature: u64::default(),
-        })
+        Ok(CheckedTransactionDetails::new(None, u64::default()))
```

Comment:
Is this `CheckedTransactionDetails::default()`?

Reply:
Yes, but that's visible only under `dev-context-only-utils`. Since this is an example for potential uses of SVM, it didn't feel right to use a DCOU API.